src/Collection/AuditedCollection.php

```diff
@@ -105,6 +105,9 @@ public function __construct(AuditReader $auditReader, $class, ClassMetadataInfo
         $this->entities = new ArrayCollection();
     }
 
+    /**
+     * @return bool
```

Comment:
Why can't we rely on the inherit doc ?

Reply:
If you don't add it here, you get deprecation from symfony. This way we silence it (and we know that at some point we will need to add it to the method directly without phpdoc.